src/components/LayoutColumns/index.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
 import {map} from 'lodash'
 
-export default ({items}) => (
-  <div className='flex-ns'>
+export default ({items, relativeSizes}) => (
+  <div className='flex-l'>
     {map(items, (item, index) => (
       <div
         key={index}
-        className={`w-100-ns ${index < items.length - 1 ? 'mr4-ns' : ''}`}
+        className={`
+          ${index < items.length - 1 ? 'mr4-l' : ''}
+        `}
+        style={{
+          flex: relativeSizes
+            ? relativeSizes[index]
+            : 1
+        }}
```

Comment:
Can now have relative sizes applied to each column:
![image](https://cloud.githubusercontent.com/assets/5497885/24429789/cb807148-13d0-11e7-9016-b43486ca7b02.png)

Can still have even columns by default:
![image](https://cloud.githubusercontent.com/assets/5497885/24429744/9b4efcf6-13d0-11e7-90f1-3ad595dd5c27.png)

Columns still stack vertically on mobile